Assignment2/TeresaGCG-TeresaGomezCarpintero.ttl

```diff
@@ -0,0 +1,9 @@
+@prefix : <http://www.example.com/> .
+
+:Class01 :includes :Sensor029 ;
+         :includes :Computer101 .
+:Sensor029 :hasMeasuremnt :Measurement8401 .
+:Measurement8401 :hasTemperature "29" ;
```

Comment:
Check other completed assignments to see how you have to provide datatypes for these literals